CHANGELOG.md

```diff
@@ -5,6 +5,10 @@
 
 The v1 release supports Cisco IOS-XR release versions from 7.7.1 to 7.9.1.
 
+### v1.1.14 (2023-11-15)
+
+- Modify the return code of 'host-check' so that 0 is returned if all checks pass, 1 otherwise (including if warnings).
```

Comment:
As mentioned in comment below - I don't _think_ this is a change in behaviour?

Regardless this should also mention that "`host-check` now summarizes failures and warnings separately" or something along those lines.